xdebug.c

```diff
@@ -1191,6 +1203,9 @@ PHP_RINIT_FUNCTION(xdebug)
 	XG(profiler_enabled) = 0;
 	XG(breakpoints_allowed) = 1;
 
+	/* Initialize gc statistics */
+	array_init(&XG(gc_runs));
```

Comment:
I tried to change this, but this hash is not internal, as its being returned from `xdebug_get_gc_stats` and used in a foreach loop for printing. I can try porting all this, but from my first understanding of `xdebug_hash` this would require a larger rewrite and a copy from `xdebug_hash` to `HashTable` in the return function.